Makefile

```diff
@@ -10,7 +10,7 @@ lint:
 	staticcheck `go list ./... | grep -Ev "(go\.dedis\.ch/dela/internal/testing|go\.dedis\.ch/dela/mino/minogrpc/ptypes)"`
 
 vet:
-	@echo "⚠️ Warning: the following only works with go >= 1.14" && \
+	@echo "⚠️ Warning: the following only works with go >= 1.19" && \
```

Comment:
It shouldn't be changed. This is related to something that `mcheck` uses that requires go >= 1.14.